src/Behat/Behat/Snippet/Printer/ConsoleSnippetPrinter.php

```diff
@@ -84,4 +91,24 @@ public function printUndefinedSteps($suiteName, array $steps)
 
         $this->output->writeln('');
     }
+
+    /**
+     * @param array<string> $usedClasses
+     */
+    public function outputClassesUsesStatements(array $usedClasses): void
+    {
+        $classCount = \count($usedClasses);
+
+        if (0 === $classCount) {
+            return;
+        }
+
+        $message = $this->translator->trans('snippet_proposal_use', ['%count%' => $classCount], 'output');
+
+        $this->output->writeln('--- '.$message.PHP_EOL);
+
+        foreach (array_keys($usedClasses) as $usedClass) {
```

Comment:
`array_keys` moved to where this method is called.

```suggestion
        foreach ($usedClasses as $usedClass) {
```